src/ConnectedMode/ServerSentEvents/SSESessionManager.cs

```diff
@@ -28,7 +28,7 @@ namespace SonarLint.VisualStudio.ConnectedMode.ServerSentEvents
     /// <summary>
     /// Reacts to project changes and opens/closes Server Sent Events sessions
     /// </summary>
-    internal interface ISSESessionManager : IDisposable
+    public interface ISSESessionManager : IDisposable
```

Comment:
Perhaps we should move the interface to Core. I'm not sure. IntegrationVSIX needs to call it so the listeners would be hooked up.

Reply:
Or `ConnectedMode`?
All of these classes are specific to Connected Mode - could they all be moved there?

It would mean `Connected Mode` would have to reference VS, but I think that's ok.

Reply:
It's already in ConnectedMode, and VSIX indeed references it now. Is that what you meant?

Reply:
Yes - put everything in `ConnectedMode`

Reply:
As discussed offline, we could have `IssueViz.Security` depend on `ConnectedMode` and move everything else out of `Core` and all of the non-packaging stuff out of `Integration.VSIX`.

Reply:
I will revisit once I implement the Taint consumer